packages/frontend/docs/Linting-and-formatting.md

```diff
@@ -0,0 +1,60 @@
+# Linting and Formatting
+For consistency and predictable organization of code, this project uses [Eslint](https://eslint.org/) and [Prettier](https://prettier.io/) for linting and formatting. 
+
+## The Setup
+The configuration for Eslint and prettier can be found at the root of this project:
+- Eslint - `.eslintrc.js`
+- Prettier - `.prettierrc`
```

Comment:
:hocho: please remove all references to Prettier in this file since we've decided to omit it for now :)